packages/components/components/elvis-toolbox/src/hooks/useLanguage.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+
+export const useLang = () => {
```

Comment:
I'd suggest using the same name for the hook and the file, and would probably not shorten `Language` to `Lang` then 😄 
```suggestion
export const useLanguage = () => {
```